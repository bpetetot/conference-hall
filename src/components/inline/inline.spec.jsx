/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import Inline from './index'

describe('components/inline', () => {
  it('should render', () => {
    expect(
      shallow(
        <Inline className="proposal-subtitle" classNameItem="proposal-subtitle-item">
          {'speakerName'}
          {'categoryLabel'}
        </Inline>,
      ),
    ).toMatchSnapshot()
  })
})
