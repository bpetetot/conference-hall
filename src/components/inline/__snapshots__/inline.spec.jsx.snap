// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/inline should render 1`] = `
<div
  className="inline-items proposal-subtitle"
>
  <span
    className="proposal-subtitle-item"
    key="0"
  >
    speakerName
  </span>
  <span
    className="inline-items-separator"
    key="1-separator"
  >
    â€¢
  </span>
  <span
    className="proposal-subtitle-item"
    key="1"
  >
    categoryLabel
  </span>
</div>
`;
